HDWalletKit/Core/Crypto/Encryption/ECDSA.swift

```diff
@@ -24,21 +24,26 @@ public final class ECDSA {
         var pKey = secp256k1_pubkey()
         
         var result = secp256k1_ec_pubkey_create(context, &pKey, prvKey)
-        if (result != 1) {
+        if result != 1 {
             return nil
         }
         
         let size = isCompression ? 33 : 65
```

Comment:
@impul 33, 65 -> think this is compressionBase

Reply:
done